assets/src/components/map/follower.tsx

```diff
@@ -21,18 +21,24 @@ import { equalByElements } from "../../helpers/array"
 import { RecenterControl } from "./controls/recenterControl"
 import { defaultCenter } from "../map"
 
-export type UpdateMapFromPointsFn = (map: LeafletMap, points: LatLng[]) => void
+export type UpdateMapFromPointsFn = (
+  map: LeafletMap,
+  points: LatLng[],
+  shouldOffset: boolean
+) => void
 
 export interface FollowerProps {
   positions: LatLng[]
   onUpdate?: UpdateMapFromPointsFn
+  shouldOffset?: boolean
 }
 
 export const Follower = ({
   positions,
   onUpdate,
   isAnimatingFollowUpdate,
   shouldFollow = true,
+  shouldOffset = true,
```

Comment:
suggestion: what do you think about doing this via currying `drawerOffsetAutoCenter` instead of adding another prop and function argument to the base signature?

Reply:
Yep! Happy to pair whenever works! I'll be back from lunch after 1300

Reply:
I did a thing: https://github.com/mbta/skate/pull/2286/commits/fe97cff54bd5c3e4ceaa8b125d372c886867847c

Reply:
Cool!
I notice that even on mobile, the stop card still is following the safe area set here 

https://github.com/mbta/skate/blob/master/assets/src/components/mapPage/mapDisplay.tsx#L675-L676

What do you think of using the `MapSafeAreaContext` to give `drawerOffsetAutoCenter` a safe bounds and then updating the `MapSafeAreaContext.Provider` value from `mapPage` based on if the drawer is open?